oap-server/banyandb-java-client/src/main/java/org/apache/skywalking/banyandb/v1/client/BulkWriteProcessor.java

```diff
@@ -52,16 +55,24 @@ protected BulkWriteProcessor(String processorName, int maxBulkSize, int flushInt
         buffer.consume(QueueWatcher.class, concurrency, 20, properties);
     }
 
+    @Override
+    public void close() throws IOException {
+        this.buffer.shutdownConsumers();
+    }
+
     /**
      * The internal queue consumer for build process.
      */
-    private static class QueueWatcher implements IConsumer {
+    public static class QueueWatcher implements IConsumer {
```

Comment:
Don't `public` this.

Reply:
> Don't `public` this.

If we don't use public, the following exception will be thrown,

```
org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumerCannotBeCreatedException: java.lang.IllegalAccessException: Class org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumeDriver can not access a member of class org.apache.skywalking.banyandb.v1.client.BulkWriteProcessor$QueueWatcher with modifiers "public"

	at org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumeDriver.getNewConsumerInstance(ConsumeDriver.java:76)
	at org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumeDriver.<init>(ConsumeDriver.java:43)
	at org.apache.skywalking.apm.commons.datacarrier.DataCarrier.consume(DataCarrier.java:103)
	at org.apache.skywalking.banyandb.v1.client.BulkWriteProcessor.<init>(BulkWriteProcessor.java:55)
	at org.apache.skywalking.banyandb.v1.client.TraceBulkWriteProcessor.<init>(TraceBulkWriteProcessor.java:53)
	at org.apache.skywalking.banyandb.v1.client.BanyanDBClient.buildTraceWriteProcessor(BanyanDBClient.java:163)
	at org.apache.skywalking.banyandb.v1.client.BanyanDBClientWriteTest.setUp(BanyanDBClientWriteTest.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at io.grpc.testing.GrpcCleanupRule$1.evaluate(GrpcCleanupRule.java:125)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.IllegalAccessException: Class org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumeDriver can not access a member of class org.apache.skywalking.banyandb.v1.client.BulkWriteProcessor$QueueWatcher with modifiers "public"
	at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
	at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(AccessibleObject.java:296)
	at java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:288)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:413)
	at org.apache.skywalking.apm.commons.datacarrier.consumer.ConsumeDriver.getNewConsumerInstance(ConsumeDriver.java:70)
	... 31 more
```